reader-gtfs/pom.xml

```diff
@@ -96,7 +88,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <!-- Currently we need a bit more memory than for other tests -->
                 <configuration>
-                    <argLine>-Xmx1200m -Xms1200m</argLine>
+                    <argLine>-Xmx1200m -Xms1200m -Duser.language=en</argLine>
```

Comment:
Why is this necessary?

Reply:
There are unittests checking error message contents which break with a different locale. We already set this in the parent POM but we need to explicitly set it here again, as we're overriding the inherited configuration.

Reply:
ok